app/views/unattended/report_templates/host_-_products.erb

```diff
@@ -0,0 +1,40 @@
+<%#
+name: Host - Products
+snippet: false
+model: ReportTemplate
+template_inputs:
+- name: Hosts filter
+  required: false
+  input_type: user
+  description: Limit the report only on hosts found by this search query. Keep empty
+    to report on all available hosts.
+  advanced: false
+  value_type: search
+  resource_type: Host
+require:
+- plugin: katello
+  version: 4.11.0
+-%>
+<%- report_headers 'Host Name', 'Organization', 'Lifecycle Environment', 'Content View', 'Host Collections', 'Virtual', 'Guest of Host', 'OS', 'Arch', 'Sockets', 'RAM', 'Cores', 'SLA', 'Role', 'Usage', 'Products', 'Last Checkin' -%>
+<%- load_hosts(search: input('Hosts filter'), includes: [:operatingsystem, :architecture, :content_view_environments, :organization, :reported_data, :subscription_facet]).each_record do |host| -%>
+<%-   report_row(
+        'Host Name': host.name,
+        'Organization': host.organization,
+        'Lifecycle Environment': host.single_lifecycle_environment,
+        'Content View': host.single_content_view,
+        'Host Collections': host_collections_names(host),
+        'Virtual': host.virtual,
+        'Guest of Host': host.hypervisor_host,
+        'OS': host.operatingsystem,
+        'Arch': host.architecture,
+        'Sockets': host.sockets,
+        'RAM': host.ram,
```

Comment:
What are the units here?

![image](https://github.com/theforeman/foreman/assets/22042343/e314302b-d605-4fdb-80cc-8b917d3267a7)

Reply:
It's in MBs seems, at least according to the fact parser. That would make sense given your values there. I can make the top bit say "RAM (MB)"

Reply:
sounds good to me! :+1: